test-results.mdx

```diff
@@ -29,14 +29,21 @@ Once tests are executed, you will receive a comment in your GitHub / Azure DevOp
 
 There will be a row for each test with the following information:
 
-| column      | description                                               |
-| ----------- | --------------------------------------------------------- |
-| description | name of the test case                                     |
-| status ✅   | execution successful                                      |
-| status ⌛️   | execution in progress                                     |
-| status ❌   | execution failed                                          |
-| status 💔   | something went wrong on our side during execution         |
-| details     | link which takes you to the test case details (see below) |
+| column      | description           |
+| ----------- | --------------------- |
+| description | name of the test case |
+| status ✅   | execution successful  |
+
+// eslint-disable-next-line prettier/prettier
```

Comment:
???

Reply:
why?

Reply:
i don't remember exactly, but with this change, the linter didn't protest. Linter forces me to do things.